README.md

```diff
@@ -0,0 +1,65 @@
+# SSS TODO_track_name
+
+This is the public repository for the **TODO** track of the Security Summer School (SSS).
+It contains information available to participants: session content, slides, activities support files.
+Apart from SSS participants, you can use this content by yourself to learn more about **TODO**.
```

Comment:
What are you supposed to replace this **TODO** with (web security, binary exploiting...)? Maybe a list of options in brackets would help the one who uses this template, to me it was unclear.

Reply:
Gotcha, I will clear that.